.github/ISSUE_TEMPLATE/challenge_proposal.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 🚀 Challenge Proposal
+about: I want to add a new challenge!
+title: ''
+labels: challenge
+assignees: ''
+
+---
+
+## Describe the challenge you'd like to add
+A clear and concise description of what you want to happen. Add any considered drawbacks.
+
+## Suggest a name
+The name of your baby. Consider that there might be adjustment in the challenge name.
+
+## Describe alternatives you've considered
+Mention all the keywords related to the development topics to be address (Adaters, RecyclerView, XML, DI, Architecture, etc..)
+
+## Describe the habilities that we should observe during the challenge resolution
```

Comment:
There is a type. I think it is abilities and not habilities

Reply:
Nice catch!